docs/book/messages.md

```diff
@@ -61,6 +61,19 @@ interface FlashMessagesInterface
      */
     public function getFlash(string $key, $default = null);
 
+    /**
+     * Retrieve all flash values.
```

Comment:
I'm not a big fan of copying code into the documentation, then we need to maintain it in two places, and always remember to update it in the docs.... I know it was already here and you've just updated it 👍